include/pm_debug.h

```diff
@@ -44,12 +44,12 @@ extern int logLevel;  //must be defined in dll and kernel object
  All we can do is write to a dedicated debug channel and adjust the loglevel at runtime.
 */
 #ifdef DEBUG_ON
-    #define DEBUG(...)  _DEBUG(0, ##__VA_ARGS__)
-    #define INFO(...)   _DEBUG(1, ##__VA_ARGS__)
-    #define WARN(...)   _DEBUG(2, ##__VA_ARGS__)
-    #define ERR(...)  _DEBUG(3, ##__VA_ARGS__)        //ERROR is already defined in wingdi.h
+    #define DEBUG(...)  DEBUG_LOG(0, ##__VA_ARGS__)
+    #define INFO(...)   DEBUG_LOG(1, ##__VA_ARGS__)
+    #define WARN(...)   DEBUG_LOG(2, ##__VA_ARGS__)
+    #define ERR(...)    DEBUG_LOG(3, ##__VA_ARGS__)        //ERROR is already defined in wingdi.h
 
-    #define _DEBUG(level, format, ...) __DEBUG(LOGGER_NAME, level, __LINE__, format, ##__VA_ARGS__)
```

Comment:
`_DEBUG` is usually reserved by the compiler to mark debug build. 

At some point we should remove all defines that start with `_` 
`external identifiers (global functions and variables) that begin with an underscore (‘_’) and all identifiers regardless of use that begin with either two underscores or an underscore followed by a capital letter are reserved names.`
https://www.gnu.org/software/libc/manual/html_node/Reserved-Names.html

Reply:
Fully agree on that part!